spec/models/user_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe "validations" do
+    it {should validate_presence_of(:name)}
+    it {should validate_presence_of(:password)}
```

Comment:
we can also add that it validates confirmation of password and has a secure password, if we wanted